internal/signalfx-agent/pkg/monitors/collectd/etcd/etcd.go

```diff
@@ -57,6 +57,7 @@ type Monitor struct {
 
 // Configure configures and runs the plugin in collectd
 func (m *Monitor) Configure(conf *Config) error {
+	m.Logger().Warn("The ectd-collectd plugin is deprecated. It will be removed in a future release.")
```

Comment:
Can you include a note about the expected migration path (including receiver name)?